apps/wallet-mobile/src/features/Receive/useCases/DescribeSelectedAddressScreen.tsx

```diff
@@ -31,6 +39,10 @@ export const DescribeSelectedAddressScreen = () => {
     copy(selectedAddress)
   }
 
+  React.useEffect(() => {
+    isSingle && isShowingMultipleAddressInfo && openModal(strings.singleOrMultiple, <Modal />, modalHeight)
```

Comment:
hey @jorbuedo by reading the ticket it looks like it should check for the used addresses and if there is more than 1  suggest the change, isSingle here is the current config only and the isShowing is for the popup, I think it needs to check for the wallet.usedAddress... to decide if whether it should open or not, can you check with Product?

Reply:
Yes, only needs to show when there's multiple used already